core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/preferences/TitleBlockPreferencesInitializer.java

```diff
@@ -19,9 +19,9 @@
 
   public static final String TABEL_CONTENT_PREFERENCE_STORE = "tableTitleBlock";

```

Comment:
This file is not ok, it rewrites malina's commit.

The correct one is with LINES_ inseatad of rows, something happened during rebase, put the correct file instead.